api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -96,7 +96,8 @@ public class MailServiceImpl implements MailService {
   private static final String WORKSPACE_ADMIN_LOCKING_RESOURCE =
       "emails/workspace_admin_locking/content.html";
 
-  private static final String PUBLISH_WORKSPACE_RESOURCE = "emails/publish_workspace/content.html";
+  private static final String PUBLISH_COMMUNITY_WORKSPACE_RESOURCE =
+      "emails/publish_community_workspace/content.html";
```

Comment:
Should we tweek the name to publish_community_workspace_per_owner or something like that to indicate we are sending emails per owners to include their name

Reply:
I agree that we want to make it obvious that we're sending multiple emails, but I'm not sure that this is the right place.  I'll think about alternatives.